config-overrides.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
This files should be removed and maybe add documentation to the readme a link how to test Safe Apps